Magento_Checkout/web/template/minicart/item/default.html

```diff
@@ -140,7 +147,14 @@ <h3 class="minicart-product__name">
                         aria-label="edit product in your shopping cart"
                     >
                         <svg class="button__icon minicart-product__button-icon">
-                            <use data-bind="attr: { 'xlink:href': require.toUrl('images/icons-sprite.svg#edit') }" xlink:href=''></use>
+                            <use
```

Comment:
please don't split to multiple lines use tag. It doesn't look ok

Reply:
BTW: here https://github.com/SnowdogApps/magento2-alpaca-components/blob/master/components/01-globals/icons/README.md is readme "How to add accessible svg icons" Please add role and title.